### ISSUE TITLE
Test build for Ubuntu Trusty x64 failed

### ISSUE THREAD

**Issue by zcsteele** [steelewool/iraf-v216#37]
The most recent test build failed for the 64-bit version of Ubuntu Trusty (14.04.5): https://travis-ci.org/steelewool/iraf-v216/jobs/236254573

None of the other build variations failed.

The errors in the build file start while compiling XPP, at line 770 in the log.

Offhand, it looks like the one or more library files may be missing but I could use a bit of help identifying the source of this issue.

**Comment by RobSteele49** [comment]
On my ubuntu 16.04 system I'm getting something similar. In my environment libslalib.a is not getting built. I grepped the spooo and spool.final files and the errors are below. I'll try and look into why this library is not being built tonight.

> steele@steele-VirtualBox:~/git/iraf-v216$
> grep collect2 spool*
> spool:collect2: error: ld returned 1 exit status
> spool.final:collect2: error: ld returned 1 exit status
> spool.final:collect2: error: ld returned 1 exit status
> steele@steele-VirtualBox:~/git/iraf-v216$ grep "cannot find" spool*
> spool:/usr/bin/ld: cannot find -lslalib
> spool.final:/usr/bin/ld: cannot find -lslalib

'/usr/bin/ld: cannot find -lslalib
collect2: error: ld returned 1 exit status
Warning, mkpkg line 7: module `relink' not found or returned error
move `xx_votools.e' to `vobin$x_votools.e'
$move: file `xx_votools.e' not found
Warning, mkpkg line 20: error moving file xx_votools.e
Warning, mkpkg line 8: module `install' not found or returned error
Warning, mkpkg line 10: module `update@votools' not found or returned error
Warning, mkpkg line 3: module `update' not found or returned error
Start:  Fri May 26 05:18:27 PDT 2017
  End:  Fri May 26 05:26:48 PDT 2017'

**Comment by RobSteele49** [comment]
When I searched the spool file I found that obs.f had an error. This was an error I've seen in the past. I'm wondering if you are seeing this same error in your build?

Compiler error line 824 of obs.f: Too many literal constants.
Table limit now 200.
Try rerunning with the -NL400 option.

sed: can't read obs.c: No such file or directory
gcc: error: obs.c: No such file or directory
gcc: fatal error: no inp

**Comment by RobSteele49** [comment]
I checked out Ole's merge-all branch and backed up till I found the last commit that did not have the error in obs.f. This was the 27166bb5ff546b6c29e7a8674bbddbb5114ba1ef commit. The commits before this all have the error.

This is the log from make sysgen |tee make-sysgen-20170527-27166bb.txt

[make-sysgen-20170526-27166bb.txt](https://github.com/steelewool/iraf-v216/files/1033314/make-sysgen-20170526-27166bb.txt)

You can search on obs.f and see what the command should be.

**Comment by RobSteele49** [comment]
The commit, 780bd19daef25ff109c7339b75b220d1723cb760, fixed this issue. But Ole needs to be informed of the fix. It's just a one line change in util/mksysgen.

**Comment by RobSteele49** [comment]
@olebole wanted to catch you up on a problem I came across in your merge-all branch. I'm compiling on a 64 bit ubuntu 16.04 machine. The last commit in your branch that would compile for me was the 27166bb5ff546b6c29e7a8674bbddbb5114ba1ef commit. The change that seems to be the problem was that one to build NOVOS was changed from cd $iraf. I did a fix in my commit https://github.com/steelewool/iraf-v216/commit/780bd19daef25ff109c7339b75b220d1723cb760 . Let me know if you see something similar when compiling the linux64 build.

**Comment by zcsteele** [comment]
@RobSteele49 I reran the failing build in Travis CI and the build (from commit 4503e35) and it succeeded this time - https://travis-ci.org/steelewool/iraf-v216/jobs/236254573

**Comment by olebole** [comment]
Can you run `diff` on the two logs and check, where exactly it failed?

**Comment by RobSteele49** [comment]
@olebole I cloned your repository this morning and did builds on the 0e60 and 2716 commits of merge-all. I saved the results of make sysgen |tee logfile and have differenced them and attached them. The errror I'm seeing in the 0e60ac19 commit is:

Compiler error line 824 of obs.f: Too many literal constants.
Table limit now 200.
Try rerunning with the -NL400 option.

Which results in the library  libslalib.a not being built.

Any thoughts you might have would indeed be appreciated. Thank.

[diff-make-sysgen.txt](https://github.com/steelewool/iraf-v216/files/1036305/diff-make-sysgen.txt)
[diff-spool.txt](https://github.com/steelewool/iraf-v216/files/1036308/diff-spool.txt)
[diff-spool.final.txt](https://github.com/steelewool/iraf-v216/files/1036311/diff-spool.final.txt)
